client/src/App.js

```diff
@@ -22,7 +23,24 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" 
+        render={props => (
+          <MovieList
+            {...props}
+            // match={props.match}
+            // location={props.location}
+            // history={props.history}
+            items={this.state.savedList}
+          />
+        )} />
+      />
+        <Route path="/movies/:id"           
+        render={props => (
+            <Movie {...props}              
+              items={this.state.savedList}
+              addToSavedList={this.addToSavedList}
```

Comment:
as written, `this.addToSavedList()` will add duplicate titles to your state.
there are a few ways to prevent this behavior, here's a code snippet that a student provided in after hours last night >> 
```
  addToSavedList = movie => {
    const savedList = this.state.savedList;
    if (!savedList.map(savedMovie => savedMovie.id).includes(movie.id)) {
      savedList.push(movie);
      this.setState({ savedList });
    }
  };
```